### DISCUSSION TITLE
When is next release planned for Organizer support?

### DISCUSSION THREAD

**Discussion by WillyReyno** [markuspoerschke/iCal#268] [Q&A] [answered]
Hello & thank you for this great package!

I noticed you guys had merged the following PR regarding Organizer support in calendars:  https://github.com/markuspoerschke/iCal/pull/260/files 

Do you know when will this be included in a release? 

Thanks again!

**Comment by markuspoerschke** [comment, marked as answer]
Thanks for the reminder.

My bot just created a new release: https://github.com/markuspoerschke/iCal/releases/tag/2.2.0

The organizer will be available in version 2.2.0.

**Reply by WillyReyno** [reply]
Thank you!
